cms/topics/models.py

```diff
@@ -1,23 +1,225 @@
     page_description = _("A specific topic page. e.g. 'Public sector finance' or 'Inflation and price indices'.")
+    label = _("Topic")
 
     summary = RichTextField(features=settings.RICH_TEXT_BASIC)
+    featured_series = models.ForeignKey(
+        "articles.ArticleSeriesPage",
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name="featured_on_topic",
+    )
+    explore_more = StreamField(ExploreMoreStoryBlock(), blank=True)
+
+    content_panels: ClassVar[list["Panel"]] = [
+        *BasePage.content_panels,
+        "summary",
+        FieldPanel(
+            "featured_series",
+            heading=_("Featured"),
+            widget=FeaturedSeriesPageChooserWidget(linked_fields={"topic_page_id": "#id_topic_page_id"}),
+        ),
+        InlinePanel(
+            "related_articles",
+            heading=_("Highlighted articles"),
+            help_text=_(
+                f"Choose up to {MAX_ITEMS_PER_SECTION} articles to highlight. "
+                "The 'Related articles' section will be topped up automatically."
+            ),
+            max_num=MAX_ITEMS_PER_SECTION,
+        ),
+        InlinePanel(
+            "related_methodologies",
+            heading=_("Highlighted methods and quality information"),
+            help_text=_(
+                f"Choose up to {MAX_ITEMS_PER_SECTION} methodologies to highlight. "
+                "The 'Methods and quality information' section will be topped up automatically."
+            ),
+            max_num=MAX_ITEMS_PER_SECTION,
+        ),
+        FieldPanel("explore_more", heading=_("Explore more")),
+    ]
+
+    search_fields: ClassVar[list[index.BaseField]] = [*BasePage.search_fields, index.SearchField("summary")]
+
+    def get_context(self, request: "HttpRequest", *args: Any, **kwargs: Any) -> dict:
+        """Additional context for the template."""
+        context: dict = super().get_context(request, *args, **kwargs)
+        context["table_of_contents"] = self.table_of_contents
+        context["featured_item"] = self.latest_article_in_featured_series
+        context["formatted_articles"] = get_formatted_pages_list(self.processed_articles, request=request)
+        context["formatted_methodologies"] = get_formatted_pages_list(self.processed_methodologies, request=request)
+        return context
+
+    @cached_property
+    def latest_article_in_featured_series(self) -> StatisticalArticlePage | None:
+        """Returns the latest article in the featured series."""
+        article: StatisticalArticlePage | None = StatisticalArticlePage.objects.none()
+        if self.featured_series:
+            article = (
+                StatisticalArticlePage.objects.child_of(self.featured_series)
+                .live()
+                .public()
+                .order_by("-release_date")
+                .first()
+            )
+        return article
```

Comment:
Nit: The type of `article` isn't quite right. The easiest fix is to just remove it.

```suggestion
        if self.featured_series:
            return (
                StatisticalArticlePage.objects.child_of(self.featured_series)
                .live()
                .public()
                .order_by("-release_date")
                .first()
            )
        return None
```

Reply:
something, something appease mypy. tweaked closer to what you suggested 069f8cd